src/wx/cmdevents.cpp

```diff
@@ -2766,10 +2769,26 @@ EVT_HANDLER_MASK(ChangeFilter, "Change Pixel Filter", CMDEN_NREC_ANY)
     }
 
     wxString msg;
-    msg.Printf(_("Using pixel filter %s"), pixel_filters->GetString(gopts.filter));
+    msg.Printf(_("Using pixel filter:") + " %s", pixel_filters->GetString(gopts.filter));
     systemScreenMessage(msg);
 }
 
+static const wxString&
+ifb_short_name(const wxChoice* ifbs, unsigned num)
+{
+    static std::map<unsigned, wxString> cache;
+
+    if (cache.count(num))
+        return cache[num];
+
+    // Shorten and clean up name.
+    cache[num] = strutils::trim(strutils::titlecase(
+        strutils::remove_word(strutils::remove_word(ifbs->GetString(num).Lower(), _("interframe")), _("blending"))
```

Comment:
There's only so many values. Could we just populate the map on first call and simply return the value on further calls? If we're doing that, we can probably use a simple `vector`.